A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_PerformancePage.js

```diff
@@ -762,19 +762,23 @@ class CDUPerformancePage {
                 CDUPerformancePage.ShowAPPRPage(mcdu);
             }
         };
-
         let transAltCell = "[\xa0][color]cyan";
-        if (isFinite(mcdu.perfApprTransAlt)) {
-            transAltCell = mcdu.perfApprTransAlt.toFixed(0);
-            if (!mcdu.perfApprTransAltPilotEntered) {
-                transAltCell += "[s-text]";
+        if (hasDestination) {
+            const arrivalTransAltitude = SimVar.GetSimVarValue("L:AIRLINER_APPR_TRANS_ALT", "Number");
+            if (isFinite(mcdu.perfApprTransAlt)) {
+                transAltCell = mcdu.perfApprTransAlt.toFixed(0);
+                if (!mcdu.perfApprTransAltPilotEntered) {
+                    transAltCell += "[s-text]";
+                }
+            } else {
+                transAltCell = "{cyan}[ ]{end}";
```

Comment:
There is no need for this, it's already done above on line 765 (except with a non-breaking space).

Reply:
It is not neex when shows blank [ ]?

Reply:
It's already set to blank at line 765, as I said. It's only changed if perfApprTransAlt is actually set.

Reply:
oh i see. i'll remove it